pkg/argo-actions/upload_type_instances.go

```diff
@@ -83,6 +85,16 @@ func (u *Upload) Do(ctx context.Context) error {
 		return errors.Wrap(err, "while rendering CreateTypeInstancesInput")
 	}
 
+	u.log.Info("Validating TypeInstances")
+
+	validationResult, err := validation.ValidateTypeInstancesToCreate(ctx, u.client, payload)
+	if err != nil {
+		return errors.Wrap(err, "while validating TypeInstances")
+	}
+	if validationResult.Len() > 0 {
+		return validationResult.ErrorOrNil()
```

Comment:
As it is difficult to verify all manifests, perhaps it would be good to print a warning and after some time create PR which will return error instead.